hw-lesson1.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+//Задание 1:
+const loop = (times = 0, cb = null) => {
+    if (cb === null) return 'Не передана функция';
+    else for (let i = 0; i < times; i++) {
+        cb();
+    }
+};
+
+const z = loop(10, () => {
+    console.log(i + 1)
```

Comment:
```suggestion
    console.log(i + 1)
```
тут ошибка "i is not defined", так как второй аргумент функции loop() имеет свой контекст выполнения в глобальном контексте, в это же время у самой функции loop() тоже свой контекст выполнения в глобальном контексте, они не пересекаются и не могут брать друг у друга данные через LHS-lookup. 
Вроде бы так можно объяснить почему не стоит использовать i во втором аргументе (в стрелочной функции) :)